### PR TITLE
Add SITE_ID var

### PR DESCRIPTION
**Pull request by AllenAnthes**

### CODE REVIEW

File: kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml [hunk 1]
```diff
@@ -7,8 +7,7 @@ spec:
     spec:
       containers:
         - name: app
-          image: operationcode/back-end:master
```
Comment (wimo7083) [inline kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml]: are you tagging your images in dockerhub as `latest` or `master`?

File: kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml [hunk 2]
```diff
@@ -7,8 +7,7 @@ spec:
...
-          imagePullPolicy: Always
```
Comment (wimo7083) [inline kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml]: I think this should stay in.

Comment (wimo7083) [reply kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml]: Okay since we are iterating we want to not pull always.

Comment (ohaiwalt) [reply kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml]: That...doesn't make sense

Comment (wimo7083) [reply kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml]: For now his development flow is to push things to master and manually deploy to staging or prod.

**Pull request merged**